result/result.mbt

```diff
@@ -296,3 +296,17 @@ pub fn unwrap[T, E](self : Result[T, E]) -> T {
     Err(_) => abort("called `Result::unwrap()` on an `Err` value")
   }
 }
+
+pub fn to_string[T : Show, E : Show](self : Result[T, E]) -> String {
+  match self {
+    Ok(x) => "Ok(\(x))"
+    Err(e) => "Err(\(e))"
+  }
+}
+
+test "show" {
+  let ok : Result[_, String] = Ok("hello")
+  inspect(ok, ~content="Ok(hello)")?
```

Comment:
`Ok(hello)` looks weird - are we sure using `Show` in `inspect()` is the right thing to do?

Reply:
it is a known issue that will be fixed cc @Guest0x0